tests/dummy/app/resolver.js

```diff
@@ -1,3 +0,0 @@
-import Resolver from "ember-resolver";
-
-export default Resolver;
```

Comment:
do you know why we had this in the first place? (cc @anehx)

Reply:
Default ember blueprint

Reply:
Ah, thanks :upside_down_face: